### DISCUSSION TITLE
warning C4307 on vs2017

### DISCUSSION THREAD

**Discussion by soncfe** [gabime/spdlog#2531] [Q&A]
Ver 1.11.0
spdlog\include\spdlog\fmt\bundled\format-inl.h(1154): warning C4307: “*”

// Remove trailing zeros from n and return the number of zeros removed (float)
FMT_INLINE int remove_trailing_zeros(uint32_t& n) noexcept {
  FMT_ASSERT(n != 0, "");
  const uint32_t mod_inv_5 = 0xcccccccd;
  const uint32_t mod_inv_25 = mod_inv_5 * mod_inv_5;//// ?????

**Comment by tt4g** [comment]
`spdlog\fmt\bundled\format-inl.h` is fmt library source code.
It is recommended to move issue to [fmt repository](https://github.com/fmtlib/fmt).
